supercommand_test.go

```diff
@@ -681,3 +681,25 @@ type flagAdderFunc func(*gnuflag.FlagSet)
 func (f flagAdderFunc) AddFlags(fset *gnuflag.FlagSet) {
 	f(fset)
 }
+
+func (s *SuperCommandSuite) TestFindClosestSubCommand(c *gc.C) {
+	sc := cmd.NewSuperCommand(cmd.SuperCommandParams{
+		UsagePrefix: "juju",
+		Name:        "command",
+		Log:         &cmd.Log{},
+	})
+	name, _, ok := sc.FindClosestSubCommand("halp")
+	c.Assert(ok, gc.Equals, true)
+	c.Assert(name, gc.Equals, "help")
+}
+
+func (s *SuperCommandSuite) TestFindClosestSubCommandReturnsExactMatch(c *gc.C) {
+	sc := cmd.NewSuperCommand(cmd.SuperCommandParams{
+		UsagePrefix: "juju",
+		Name:        "command",
+		Log:         &cmd.Log{},
+	})
+	name, _, ok := sc.FindClosestSubCommand("help")
+	c.Assert(ok, gc.Equals, true)
+	c.Assert(name, gc.Equals, "help")
+}
```

Comment:
can we have a test case for something that's off the wall and won't match please?

Reply:
of course...